Makefile

```diff
@@ -152,9 +152,9 @@ test-e2e-ci: ## Run the end-to-end tests (used in the CI)`
 
 .PHONY: test-book
 test-book: ## Run the cronjob tutorial's unit tests to make sure we don't break it
-	cd ./docs/book/src/cronjob-tutorial/testdata/project && make test
-	cd ./docs/book/src/component-config-tutorial/testdata/project && make test
-	cd ./docs/book/src/multiversion-tutorial/testdata/project && make test
+	cd ./docs/book/src/cronjob-tutorial/testdata/project && rm -rf test && make test
+	cd ./docs/book/src/component-config-tutorial/testdata/project && rm -rf test && make test
+	cd ./docs/book/src/multiversion-tutorial/testdata/project && rm -rf test && make test
```

Comment:
Why did you remove the tests from here?
Is the make test calling the new e2e tests?
If so, the best solution is not call the e2e tests in the make test

Reply:
yeah, whenever make test is called (https://github.com/kubernetes-sigs/kubebuilder/blob/master/testdata/project-v4/Makefile#L65 )
at the end of the line there is go test ./... which will run the go test in every directory including the e2e dir where test suite is present